file/file_util.cpp

```diff
@@ -25,7 +25,7 @@
 #include "file/file_util.h"
 #include "util/text/utf8.h"
 
-#if defined(__FreeBSD__) || defined(__APPLE__)
+#if !defined(__linux__) && !defined(__SYMBIAN32__)
```

Comment:
I'd prefer consistency to avoid confusion in future. Does it need a new pull request or can be cherry-picked from cbea18a ?